packages/http/src/__tests__/fixtures/server-validation.oas3.json

```diff
@@ -1,5 +1,5 @@
 {
-  "swagger": "3.0.0",
+  "openapi": "3.0.0",
```

Comment:
Yeah, this test was broken because this is an invalid OpenAPI document. I fixed it.